modules/hcp-adding-agents.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-adding-agents_{context}"]
+= Adding agents to the heterogeneous cluster
+
+You can add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.
```

Comment:
If this task is required, you might remove the _can_ here, as it makes it sound like the task is optional. 

```suggestion
You add agents by manually configuring the machine to boot with a live ISO. You can download the live ISO and use it to boot a bare-metal node or a virtual machine. On boot, the node communicates with the `assisted-service` and register as an agent in the same namespace as the `infraenv`. Once each agent is created, you can optionally set its `installation_disk_id` and `hostname` parameters in the specifications. When you are done, approve it to indicate that the agent is ready for use.
```